api/anri/apps/products/models/product.py

```diff
@@ -8,5 +8,7 @@ class Product(CoreModel):
     code = models.CharField(max_length=127)
     group = models.CharField(max_length=127)
     description = models.CharField(max_length=127)
```

Comment:
Description could be longer. Increase it to 512 characters or use dynamic length type